static/js/dashboard-logs-results-grid.js

```diff
@@ -112,6 +112,10 @@ function getPanelGridOptions() {
 
 
 function renderPanelLogsGrid(columnOrder, hits, panelId,currentPanel) {
+    if (typeof panelId === 'undefined' || panelId === null) {
```

Comment:
<img width="1417" alt="image" src="https://github.com/siglens/siglens/assets/71771131/67c9ec5d-d441-4a37-b658-18f7c1e9bc10">
Still shows an error when we scroll.